avm/src/daemon/daemon.rs

```diff
@@ -93,6 +97,24 @@ async fn post_v1_scale(cluster_id: &str, agz_b64: &str, deploy_token: &str, fact
   post_v1("scale", scale_body).await
 }
 
+async fn get_procs_cpu_usage() -> Vec<f32> {
+  let mut usages = Vec::new();
+  let mut processes = processes();
+  while let Some(process) = processes.next().await {
+    if let Ok(proc) = process {
+      let measurement_1 = proc.cpu_usage().await.expect("Failed to get CPU usage for process");
+      // CpuUsage struct represents instantaneous CPU usage and
+      // does not represent any reasonable value by itself
+      sleep(Duration::from_secs(1)).await;
```

Comment:
@dfellis thoughts on how long we should wait here? They recommend an absolute minimum of 100ms in the documentation.

Reply:
Well, a 1 sec wait is alright, but doing it sequentially like this doesn't seem good, since the time it takes to run this function will vary depending on how many processes the OS is running. Can you put the body of this loop inside of an `async move` instead and then execute them in parallel?

Reply:
I know. It seems like we can't :'( Streams of futures can't be iterated over and the only concurrent method they have doesn't allow mutations. Hmm unless we put it in an Arc, let me see. https://docs.rs/futures-preview/0.2.2/futures/stream/trait.StreamExt.html#method.for_each_concurrent

Reply:
That can also be in a followup PR though so that I unblock you

Reply:
Followup PR sounds good to me.